tests/test_wiktionary_definitions.py

```
@@ -0,0 +1,269 @@
+    r")",
+    re.IGNORECASE,
+)
+
+# Section header pattern
+SECTION_HEADER = re.compile(r"^={2,4}\s*.*={2,4}\s*$")
+
+# Phonetic transcription patterns
+PHONETIC_PATTERN = re.compile(r"^[/\\\[]")
+
+
+def is_quality_definition(word, lang_code, definition):
```

Ruff (0.15.2)

[warning] Unused function argument: `lang_code`

(ARG001)